docs/Updating-v0.3.md

```diff
@@ -0,0 +1,17 @@
+# Updating from v0.2 to v0.3 of ML-Agents
+
+## Important
+ * ML-Agents will no longer support Python 2. 
+
+## Training
+ * The script `learn.py` now replaces `ppo.py` and the `PPO.ipynb` python notebook.
+ * Training hyperparameters are now given to trainers via the `trainer_config.yaml` file.
+
+## Environment
+ * Modifications to an Agent's rewards must be done via `AddReward()` or `SetReward()`.
+ * Setting an Agent to done now requires the use of the `Done` method.
+ * `CollectStates()` is replaced by `CollectObservations()`.
+ * To collect observations, call `AddVectorObs()` within `CollectObservations()`.
```

Comment:
Make clearer that there are multiple versions of AddVectorObs

Reply:
And that you pass floats/ints/arrays/lists to it in order to add them to the agent's observation.